schemas/rum/view-schema.json

```
@@ -313,6 +313,30 @@
               "description": "Version of the update of the view event",
               "minimum": 0,
               "readOnly": true
+            },
+            "page_states": {
+              "type": "array",
+              "description": "List of the page states during the view",
+              "items": {
+                "type": "object",
+                "description": "Properties of the page state",
+                "required": ["state", "start"],
+                "properties": {
+                  "state": {
+                    "enum": ["active", "passive", "hidden", "frozen", "terminated"],
+                    "description": "Page state name",
+                    "readOnly": true
+                  },
+                  "start": {
+                    "type": "integer",
+                    "description": "Duration in ns to the start of the page state",
```

Just clarify the doc and match the `in_foreground_periods.start` attribute.
```suggestion
                    "description": "Duration in ns between start of the view and start of the page state",
```